### PR TITLE
docs: Rename Best Practiced to Guides; Add Try Out with Gitpod

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**
- Add Terminology page
- Rename **Best Practices** to **Guides**
- Fix Heading for Tooling -> Annotation
- Add a card choice for either **Try Out with Gitpod** or **Install on local machine**

### CODE REVIEW

File: docs/docs/general/08-Terminology.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+| Term                   | Meaning     |
+| ---------------------- | ----------- |
+| annotation             | Marked area around an element with metadata name, text, and bounding box coordinates. |
+| access token           | Gives you access to the askui inference in combination with your `workspace id`. Every `access token` has an expiry date and is assigned to exactly one workspace.  |
+| automation             | A set of instructions to achieve a task. |
+| bounding box           | Rectangle described by coordinates that describe an element's location. Displayed visually as red rectangle. |
+| command                | An action on the user interface. For example mouse left click or keypress. |
+| credentials            | Combination of `workspace id` and `access token` living in the same workspace. |
+| element                | Or user interface element. Building blocks websites or apps are build from. |
+| filters                | Type of an element. For example _button_. |
```
Comment (thinh-askui) [inline docs/docs/general/08-Terminology.md:10]:
```suggestion
| filters                | Screen for a type of an element. For example _button_. |
```

File: docs/docs/general/08-Terminology.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+| Term                   | Meaning     |
+| ---------------------- | ----------- |
+| annotation             | Marked area around an element with metadata name, text, and bounding box coordinates. |
+| access token           | Gives you access to the askui inference in combination with your `workspace id`. Every `access token` has an expiry date and is assigned to exactly one workspace.  |
+| automation             | A set of instructions to achieve a task. |
+| bounding box           | Rectangle described by coordinates that describe an element's location. Displayed visually as red rectangle. |
+| command                | An action on the user interface. For example mouse left click or keypress. |
+| credentials            | Combination of `workspace id` and `access token` living in the same workspace. |
+| element                | Or user interface element. Building blocks websites or apps are build from. |
```
Comment (thinh-askui) [inline docs/docs/general/08-Terminology.md:9]:
```suggestion
| element                |  A user interface component which websites and blocks are build from. |
```

**Review by thinh-askui — APPROVED** [review]
Looking really good. I have only some minor remarks

File: docs/docs/general/08-Terminology.md (hunk 1 above)
Comment (JohannesDienst-askui) [reply docs/docs/general/08-Terminology.md:10]: I see why you made the change but I do not understand what **Screen** means here 😄 . @thinh-askui would you explain, please?

File: docs/docs/general/08-Terminology.md (hunk 2 above)
Comment (JohannesDienst-askui) [reply docs/docs/general/08-Terminology.md:9]: Much simpler! Great.

File: docs/docs/general/02-Getting Started/gitpod-try-out.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+sidebar_position: 4
+---
+
+# Try Out with Gitpod
```
[the remark this answers is not in the document]
Comment (JohannesDienst-askui) [reply docs/docs/general/02-Getting Started/gitpod-try-out.md:5]: Yes you are right. I even forgot that I added this file 😨

File: docs/docs/general/08-Terminology.md (hunk 1 above)
Comment (thinh-askui) [reply docs/docs/general/08-Terminology.md:10]: Sorry, should be *screens. First I wanted to write "Filters for a type of an element."
Idea was just to use a synonym for filter. Maybe it's not the best synonym :D

Comment (JohannesDienst-askui) [reply docs/docs/general/08-Terminology.md:10]: We can go with **screens**.

**Pull request merged**